epoxy-processor/src/main/java/com/airbnb/epoxy/GeneratedModelWriter.java

```diff
@@ -441,7 +441,8 @@ private void addDefaultLayoutMethodIfNeeded(TypeElement originalClassElement,
       return;
     }
 
-    AndroidResource layoutResource = resourceProcessor.getResourceForValue(layoutRes);
+    AndroidResource layoutResource = resourceProcessor.getResourceForValue(
```

Comment:
since this is the only place that `getResourceForValue` and `elementToQualifiedId` are called it seems like we could combine them into one method that returns `AndroidResource` and never expose the qualified id here. possibly just changing the parameters of `getResourceForValue` to `(originalClassElement, layoutRes)`, but maybe there is a better name